app/forms/candidate_interface/course_selection/course_site_step.rb

```diff
@@ -6,7 +6,7 @@ class CourseSiteStep < DfE::Wizard::Step
       validates :course_option_id, presence: true
 
       def self.route_name
-        'candidate_interface_continuous_applications_course_site'
+        'candidate_interface_course_choices_course_site'
       end
```

Comment:
In an ideal world the name of the routes should match the name of the modules and DfE wizard won't need this many route names. I think this is kinda of a tech debt 🤔 🤔 🤔

Reply:
I'll go back and change this in one of the following PRs 👌
I should have noted this